miniapps/blocksolvers/div_free_solver.hpp

```diff
@@ -0,0 +1,252 @@
+#include "mfem.hpp"
```

Comment:
Adding the mentioned references (in the PR) for the method as a comment in the beginning of the file would be useful.


    Vassilevski, (Appendix F.3) Multilevel Block Factorization Preconditioners, Springer, 2008.
    Voronin, Lee, Neumüller, Sepúlveda, Vassilevski, Space-time discretizations using constrained first-order system least squares (CFOSLS). J. Comput. Phys. 373: 863-876, 2018.